app/src/components/utils.js

```diff
@@ -50,47 +50,67 @@ const utils = {
    */
   validateKeys: obj => {
     Object.keys(obj).forEach(k => {
-      if (obj[k] === Object(obj[k]))  {
+      if (obj[k] === Object(obj[k])) {
         return utils.validateKeys(obj[k]);
       }
       if (!/^\w+$/.test(k)) throw new Error(`Invalid field name (${k}) in \`context\`.  Only alphanumeric characters and underscore allowed.`);
     });
     return true;
   },
 
+  /** Inspects an array of attachments for validity, otherwise throws an exception with the validation error
+   * @param {object[]} attachments An array of attachment items
```

Comment:
Most of the other @param etc seem to have 2 spaces after the *

Reply:
Slipped by - fixed